dsf-gdb/org.eclipse.cdt.dsf.gdb.ui/META-INF/MANIFEST.MF

```diff
@@ -3,7 +3,7 @@ Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-Vendor: %providerName
 Bundle-SymbolicName: org.eclipse.cdt.dsf.gdb.ui;singleton:=true
-Bundle-Version: 2.8.500.qualifier
+Bundle-Version: 2.8.600.qualifier
```

Comment:
This version bump is unneeded because it has already been bumped since last released. If you were getting an error without this bump, please let me know as that indicates a build script error.

```suggestion
Bundle-Version: 2.8.500.qualifier
```